src/applications/discharge-wizard/components/v2/resultsComponents/StepTwo.jsx

```diff
@@ -174,20 +231,14 @@ const StepTwo = ({ formResponses }) => {
           )}
         </li>
         {renderMedicalRecordInfo(formResponses)}
-        <li>
-          <strong>“Buddy Statements” or Other References From Service</strong>:
-          On top of military records, you can attach statements from friends or
-          colleagues you knew while in the service, or other individuals with
-          direct knowledge of your time in the military. The content of the
-          letter is more important than who it comes from, as long as the
-          writer’s opinion is credible and they know you well. The writer should
-          state how they learned about the facts or opinions they’re writing
-          about. The letters may include statements about your achievements in
-          the military, positive relationships you formed in the military, why
-          your discharge may be unjust or incorrect, and your good deeds during
-          that time.
-        </li>
-        <li>
+        {renderBuddyStatements()}
+        {renderTestatments()}
+        {/* <li>
```

Comment:
Sorry, I just realized I'm doing these as single comments and spamming your email rather than including them all together. But one more - realized this was commented out - is it needed?

And also noticed there's a typo in `renderTestaments` above.